Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Text;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887)